frontend/src/__tests__/cypress/cypress/pages/modelServing.ts

```diff
@@ -116,35 +116,7 @@ class ModelServingGlobal {
 
 class ServingRuntimeGroup extends Contextual<HTMLElement> {}
 
-class InferenceServiceModal extends Modal {
-  k8sNameDescription = new K8sNameDescriptionField('inference-service');
-
-  constructor(private edit = false) {
-    super(`${edit ? 'Edit' : 'Deploy'} model`);
-  }
-
-  findConnectionType(name: string | RegExp) {
-    return this.findExistingConnectionSelect()
-      .findByRole('button', { name: 'Typeahead menu toggle' })
-      .findSelectOption(name);
-  }
-
-  findSubmitButton() {
-    return this.findFooter().findByTestId('modal-submit-button');
-  }
-
-  findModelNameInput() {
-    return this.k8sNameDescription.findDisplayNameInput();
-  }
-
-  findSpinner() {
-    return this.find().findByTestId('spinner');
-  }
-
-  findServingRuntimeSelect() {
-    return this.find().findByTestId('inference-service-model-selection');
-  }
-
+class ServingModal extends Modal {
```

Comment:
Yeah this diff looks weird. The stuff above was not removed, but I did create a new "base" ServingModal to include anything with the template selection since it's shared in both multi-model modal and single-model modal